cockroachcloud/cloud-sso-sql.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Cluster Single Sign-on (SSO) for CockroachDB Cloud
+summary: Overview of Cluster Single Sign-on (SSO) for {{ site.data.products.db }}, review of authenticating users, configuring required cluster settings.
+toc: true
+docs_area: manage
+---
+
+Cluster SSO allows users to access the SQL interface of a CockroachDB cluster (whether provisioned on {{ site.data.products.db }} or self-hosted) with the full security of Single Sign-On (SSO), and the convenience of being able to choose from a variety of SSO identity providers, including {{ site.data.products.db }}, Google, Azure, GitHub, or your own self-hosted OIDC.
+
+This page describes:
+
+- [Authenticating human users](#authenticate-human-users-with-the-cloud-console) with the {{ site.data.products.db }} Console acting as identity provider (IdP) or token issuer
+- [Authenticating service accounts](#authenticate-application-users-service-accounts) with an external token issuer
+- [Configuring the cluster settings](#configure-your-cluster-to-accept-your-external-identity-provider) required to authenticate service accounts
```

Comment:
Thinking we could probably have these sections on a different page since this page is about Cloud.